src/test/ui/const-generics/const-param-type-depends-on-const-param.stderr

```diff
@@ -0,0 +1,24 @@
+error[E0769]: the type of const parameters must not depend on other generic parameters
+  --> $DIR/const-param-type-depends-on-const-param.rs:9:52
+   |
+LL | pub struct Dependent<const N: usize, const X: [u8; N]>([(); N]);
+   |                                                    ^ const parameters must have a concrete type
```

Comment:
Perhaps it would make more sense to make this something like:
```
^ the type must not depend on the parameter `N`
```
so it's clear why `N` is being pointed to here. ("const parameters must have a concrete type" sounds fine, but I would expect the span to be on `X` in that case, or the entire type of `X`.)

Reply:
Updated the help message